homework-submission/todo_react/src/TodoLists.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./TodoLists.css";
+
+
+
+export default class TodoLists extends React.Component {
+
+  render() {
+
+    let { description, deadline, done, id, handleClick, removeItem } = this.props;
+
+    // console.log(data[2].deadline)
```

Comment:
as a previous comment.
No console logs or un-needed comments in your currant or future projects